build/package.proj

```diff
@@ -44,6 +47,38 @@
 
     <MSBuild Projects="@(WebSdkPackageProject)"
          Targets ="Restore;Build;Pack"/>
-    
+
+  </Target>
+
+  <Target Name="_GenerateBundledVersionsProps">
```

Comment:
I do not think you should create a bundled versions file in this repo.  I think you should just hard code a default for the version in Sdk.DefaultItems.targets, if it's not already set.  Then update the bundled versions file in dotnet/cli to include the ASP.NET Core version.

If you put this version in this repo you will have to update the web SDK every time there's a new version of ASP.NET Core.  That will probably be hard to sync when shipping preview and patch versions of ASP.NET Core.

Reply:
My understanding from the dotnet sdk is to set the versions here for testing and then allow CLI to override. However, now that I think about it, our unit tests will be in a different repo and I don't think there will be anything broken here without default versions. I'll check to make sure but maybe we can remove this component.